pytest_github_actions_annotate_failures/plugin.py

```diff
@@ -1,12 +1,21 @@
 # -*- coding: utf-8 -*-
-from __future__ import print_function
+
+from __future__ import annotations, print_function
```

Comment:
FYI, `print_function` can be removed, this requires 3.7+ now.

Reply:
Thanks for the tip. Removed it: https://github.com/pytest-dev/pytest-github-actions-annotate-failures/pull/70/commits/cdb6f4f7830b8e47e1b35a7d1f32a59ef8c7f7b6